types/googlepay/index.d.ts

```diff
@@ -1227,7 +1231,7 @@ declare namespace google.payments.api {
      * - `PAYPAL`:
      *   PAYPAL payment method.
      */
-    type PaymentMethodType = "CARD" | "PAYPAL";
+    type PaymentMethodType = 'CARD' | 'PAYPAL';
```

Comment:
Can we keep these string values as double quotes as they are code generated with double quotes.